src/core/positioning/positioning.h

```diff
@@ -215,6 +217,16 @@ class Positioning : public QObject
      */
     void setLogging( bool logging );
 
+    /**
+     * Returns TRUE if the background mode is active.
+     */
+    bool backgroundMode() const;
+
+    /**
+     * Sets whether the background mode is active.
```

Comment:
Let's not state the obvious, instead: what is background mode?

Reply:
@m-kuhn , when background mode is turned on, positioning information signals are not emitted, instead the positions are saved into a file collecting position. 

This is a skeleton of a framework needed for background tracking support.